src/components/GenreSelect.tsx

```diff
@@ -0,0 +1,38 @@
+import { Component } from "react";
+import styles from "./GenreSelect.module.scss";
+
+interface GenreSelectProps {
+	genres: string[];
+	selectedGenre: string | null;
+	onSelect: (genre: string) => void;
+}
+
+export default class GenreSelect extends Component<GenreSelectProps> {
+	getButtonClasses(genre: string): string {
```

Comment:
Here is what I mean. Better not to construct custom styles for each button in project but to create <Button> component  and pass two props. "type" and "selected"